applications/openshift/etcd/etcd_peer_auto_tls/oval/shared.xml

```diff
@@ -23,7 +23,7 @@
   </ind:textfilecontent54_object>
 
   <ind:textfilecontent54_state id="state_etcd_peer_auto_tls" version="1">
-    <ind:subexpression datatype="string" operation="pattern match">^true$</ind:subexpression>
+    <ind:subexpression datatype="string" operation="pattern match">^false$</ind:subexpression>
```

Comment:
The original definition seems correct to me.
Note that the test is negated in line 9, so the definition evaluates to `pass` when `ETCD_PEER_AUTO_TLS` is not `true`.

Technically that would allow `ETCD_PEER_AUTO_TLS` to have another other value, like `maybe`.
If  `ETCD_PEER_AUTO_TLS` is a boolean setting I don't see any practical difference between the approaches.
But personally, reading the definition without negation is easier, so I'd suggest to remove the `negate="true"` from the `criterion`.

The negation was introduced in https://github.com/ComplianceAsCode/content/commit/c0ea52c56a07b95c0d740781c9719b414f2a9de3.

Reply:
OK same as above